video_emotion/emotionTagger/t/test_face_emotion_extraction.py

```diff
@@ -6,57 +6,44 @@
 from video_emotion.emotionTagger.face_emotion_extraction import classify_faces
 from video_emotion.emotionTagger.face_emotion_extraction import get_labels
 from video_emotion.emotionTagger.face_emotion_extraction import preprocess_input
```

Comment:
I think according to PEP8 we should find a way to write our imports in less than 79 chars.

Reply:
:+1: to Casper's comment